pdns/recursor_cache.cc

```diff
@@ -310,6 +302,7 @@ void MemRecursorCache::replace(time_t now, const DNSName &qname, const QType& qt
     }
   }
   ce.d_records.clear();
+  ce.d_bytes=sizeof(ce)+qname.wirelength();
 
```

Comment:
In addition to `sizeof(ce)` we need to consider the cost of the indexes. Last time I checked (not recently I'm afraid), the cost for one additional entry was:
* ordered: 3 pointers
* unordered: 2 pointers
* sequenced: 2 pointers

Reply:
It might even be implementation dependent, but this is exactly the kind of stuff I put in the NSEC(3) byte count based on some quick googling. For ce/nce I'm pondering leaving it to `*-entries` to map to this overhead but it may not be what people expect.